Options/Applicative.hs

```diff
@@ -186,33 +192,30 @@ module Options.Applicative (
   columns,
   defaultPrefs,
 
-  -- * Types
-  ParseError(..),
-  ParserFailure(..),
-  ParserResult(..),
-  CompletionResult(..),
-
-  -- * Internal
-
-  -- ** Completion Builders
+  -- * Completions
+  --
+  -- | optparse-applicative supplies a rich completion system for bash,
+  -- zsh, and fish shells.
+  --
+  -- 'Completer' functions are used for option and argument to complete
+  -- their values.
+  --
+  -- Use the 'completer' builder to use these.
+  -- The 'action' and 'completeWith' builders are also provided for
+  -- convenience, to use 'bashCompleter' and 'listCompleter' as a 'Mod'.
   Completer,
   mkCompleter,
   listIOCompleter,
+
   listCompleter,
   bashCompleter,
 
-  -- ** Parsers Runners
-  runParserInfo,
-  runParserFully,
-  runParser,
-  evalParser,
-
-  -- ** Low-level utilities
-  mapParser,
-  treeMapParser,
-  optionNames,
-  liftOpt,
-  showOption,
+  -- * Types
+  ParseError(..),
+  ParserHelp(..),
```

Comment:
I've added `ParserHelp` to main module export list as it's required for `overFailure` to be useful.